### PR TITLE
[C++] Update drogon

### PR DESCRIPTION
**Pull request by waghanza**

### CODE REVIEW

**Comment by waghanza** [comment]
seems that we return html instead of plain text @an-tao 
https://github.com/the-benchmarker/web-frameworks/actions/runs/4477581554/jobs/7869300520?pr=6234#step:8:7

**Comment by an-tao** [comment]
> seems that we return html instead of plain text @an-tao https://github.com/the-benchmarker/web-frameworks/actions/runs/4477581554/jobs/7869300520?pr=6234#step:8:7

Hi, have you changed your testing requirements? Can you give me some documentation on the modification?

**Comment by waghanza** [comment]
No changes was made @an-tao

**Comment by waghanza** [comment]
@an-tao Could you take a look. Every URL is OK, except `/` that lead to a 404
```sh
curl -v http://172.17.0.2:3000
```

```
*   Trying 172.17.0.2:3000...
* Connected to 172.17.0.2 (172.17.0.2) port 3000 (#0)
> GET / HTTP/1.1
> Host: 172.17.0.2:3000
> User-Agent: curl/7.82.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< content-length: 571
< content-type: text/html; charset=utf-8
<
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white" text="black">
<center><h1>404 Not Found</h1></center>
<hr><center>drogon/1.8.4</center>
</body>
</html>
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
* Connection #0 to host 172.17.0.2 left intact
*
```

**Comment by an-tao** [comment]
@waghanza Thanks for the feedback, this seems to be a bug introduced with version 1.8.4. I tested the latest version and it works fine. Sorry for confusing you.

File: cpp/drogon/config.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ framework:
     - build/drogon_benchmark
 
   download:
-    - git clone --branch v1.8.3 https://github.com/an-tao/drogon
+    - git clone --branch v1.8.4 https://github.com/drogonframework/drogon
```
Comment (waghanza) [inline cpp/drogon/config.yaml:25]:
```suggestion
    - git clone --branch v1.8.5 https://github.com/drogonframework/drogon
```

**Review by an-tao — APPROVED** [review]

**Pull request merged**